### DISCUSSION TITLE
Error message I cannot find a solution for

### DISCUSSION THREAD

**Discussion by Davial2** [Tensai75/nzb-monkey-go#25] [Q&A] [answered]
Hey Tensai ,

Thanks for your awesome program!

For some NZBLNKs, I receive the following error:

Error while scanning group 'xyz' for the first message: end date of search range is older than oldest message of this group

I cannot find anything concerning this problem. Only a fix by you in a previous version for this error but with "... newer than ...".

Is it about the retention my provider has? What does this error mean?

Thx :))

**Comment by Tensai75** [comment, marked as answer]
> Is it about the retention my provider has?

Most probably yes. As the error states, the oldest available message on your providers NNTP server in this particular group is younger than the time period you are are searching in. So obviously the upload you are looking for is outside the retention of your provider.

Regards, Tensai
